examples/Cassie/run_osc_jumping_controller.cc

```diff
@@ -0,0 +1,412 @@
+                                          land_time};
+
+  Vector3d support_center_offset;
+  support_center_offset << FLAGS_x_offset, 0.0, 0.0;
+  std::vector<double> breaks = com_traj.get_segment_times();
+  VectorXd breaks_vector = Eigen::Map<VectorXd>(breaks.data(), breaks.size());
+  MatrixXd offset_points = support_center_offset.replicate(1, breaks.size());
+  PiecewisePolynomial<double> offset_traj =
+      PiecewisePolynomial<double>::ZeroOrderHold(breaks_vector, offset_points);
+  com_traj = com_traj + offset_traj;
+
+  MatrixXd pos_cov = MatrixXd::Zero(nq, nq);
+  pos_cov(4, 4) = 0.0;
+  MatrixXd vel_cov = MatrixXd::Zero(nv, nv);
+  vel_cov(5, 5) = 0.0;
+
+  /**** Initialize all the leaf systems ****/
+  SIMULATOR simulator;
+  simulator = DRAKE;  // Default is DRAKE
+  if (FLAGS_simulator == "MUJOCO") simulator = MUJOCO;
+
+  drake::lcm::DrakeLcm lcm;
+
+  vector<pair<const Vector3d, const Frame<double>&>> contact_points;
+  contact_points.push_back(LeftToeFront(plant_wo_springs));
+  contact_points.push_back(RightToeFront(plant_wo_springs));
+  contact_points.push_back(LeftToeRear(plant_wo_springs));
+  contact_points.push_back(RightToeRear(plant_wo_springs));
+
+  auto state_receiver =
+      builder.AddSystem<systems::RobotOutputReceiver>(plant_w_springs);
+  auto com_traj_generator = builder.AddSystem<COMTrajGenerator>(
+      plant_w_springs, contact_points, com_traj, FLAGS_delay_time);
+  auto l_foot_traj_generator = builder.AddSystem<FlightFootTrajGenerator>(
+      plant_w_springs, "hip_left", true, l_foot_trajectory, FLAGS_delay_time);
+  auto r_foot_traj_generator = builder.AddSystem<FlightFootTrajGenerator>(
+      plant_w_springs, "hip_right", false, r_foot_trajectory, FLAGS_delay_time);
+  auto pelvis_rot_traj_generator =
+      builder.AddSystem<PelvisOrientationTrajGenerator>(
+          plant_w_springs, pelvis_rot_trajectory, "pelvis_rot_tracking_data",
+          FLAGS_delay_time);
+  auto fsm = builder.AddSystem<dairlib::examples::JumpingEventFsm>(
+      plant_w_springs, transition_times, FLAGS_contact_based_fsm,
+      FLAGS_transition_delay, (FSM_STATE)FLAGS_init_fsm_state, simulator);
+  auto command_pub =
+      builder.AddSystem(LcmPublisherSystem::Make<dairlib::lcmt_robot_input>(
+          FLAGS_channel_u, &lcm, TriggerTypeSet({TriggerType::kForced})));
+  auto command_sender =
+      builder.AddSystem<systems::RobotCommandSender>(plant_w_springs);
+  auto osc = builder.AddSystem<systems::controllers::OperationalSpaceControl>(
+      plant_w_springs, plant_wo_springs, true,
+      FLAGS_print_osc); /*print_tracking_info*/
+  auto osc_debug_pub =
+      builder.AddSystem(LcmPublisherSystem::Make<dairlib::lcmt_osc_output>(
+          "OSC_DEBUG", &lcm, TriggerTypeSet({TriggerType::kForced})));
+  auto gaussian_noise = builder.AddSystem<systems::GaussianNoisePassThrough>(
+      plant_w_springs.num_positions(), plant_w_springs.num_velocities(),
+      plant_w_springs.num_actuators(), pos_cov, vel_cov);
```

Comment:
I think we can probably move these three lines, `pos_cov`, `vel_cov`, and line 344-345 inside the if statement at line 347?
To be clearer, move them into 
```
  if (FLAGS_add_noise) {
    std::cout << "Running with noise: " << std::endl;
    controller_state_input = gaussian_noise;
  }
```